### ISSUE TITLE
gradlew.bat app:properties

### ISSUE THREAD

**Issue by HennieReyneke** [luanpotter/bgug#3]
**These are the steps followed in Android Studio Terminal:**
- flutter pub get
- flutter packages pub run build_runner build --delete-conflicting-outputs
- flutter run (on a real device)

**This is the output:**

Resolving dependencies...

* Error running Gradle: ProcessException: Process "C:\Users\hhh\AndroidStudioProjects\bgug\android\gradlew.bat" exited abnormally: Command: C:\Users\hhh\AndroidStudioProjects\bgug\android\gradlew.bat app:properties

Please review your Gradle project setup in the android/ folder.

**Comment by luanpotter** [comment]
Gradle logs should tell you what went wrong. The line you provided is just the final message, you need to provide the whole log for me to know what is up.

I will make a wild guess and say you have the following error:

```                                                       
* What went wrong:                                                      
A problem occurred evaluating project ':app'.                           
> $HOME/bgug/android/keys/key.properties (No such file or directory)
```

Which means you have not setup the android key. You need to create a pair of public/private keys for your project for gradle to sign the project. These keys are not committed to the repo, so you need your own. Or alter the build dot gradle file to skip signing.

**Comment by HennieReyneke** [comment]
Thanks for the reply, Luan. Tried skipping signing, but getting the same issue. I've tried to get the gradle logs, but was not successful. How can I get them please? Hennie

**Comment by luanpotter** [comment]
have you tried skipping by comment our the gralde.build lines related to that? there several lines to comment.

About the logs, I think there is a verbose or debug mode on gradle to get more info.

are you able to run other flutter projects normally?

**Comment by HennieReyneke** [comment]
- Yes, I commented out most of the stuff in signingConfigs. Followed some posts about disabling signing, but it didn't seem to do anything. 
- Yes, other Flutter projects run normally.
- In Android Studio, there is LogCat, where one can filter the logs. I'm not seeing anthing Gradle there though.

**Comment by luanpotter** [comment]
This was fixed in #4, there were lots of problems that were introduced as new versions of flutter and flame came along. The current version is working on android and iOS with the latest versions.
